documentation/source/user_section/installation.rst

```diff
@@ -80,9 +80,92 @@ recommend you download the Github version.
 Install on Windows 10 with WSL
 ------------------------------
 
-Windows subsystem for Linux (WSL) is available on Windows 10 and it makes it possible to run native Linux programs,
-such as SCT. Checkout the `installation tutorial for WSL <https://github.com/neuropoly/spinalcordtoolbox/wiki/SCT-on-Windows-10:-Installation-instruction-for-SCT-on-Windows-subsytem-for-linux>`_.
+Windows subsystem for Linux (WSL) is available on Windows 10 and it makes it possible to run native Linux programs, such as SCT.
 
+#. Install Windows Subsystem for Linux (WSL)
+
+   - Install `Xming <https://sourceforge.net/projects/xming/>`_.
+
+   - Install  `Windows subsystem for linux and initialize it <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_.
+
+       - Please install the Ubuntu 18.04 LTS distro.
+
+#. Environment preparation
+
+   Run the following command to install various packages that will be needed to install FSL and SCT. This will require your password
+
+   .. code-block:: sh
+
+      sudo apt-get -y update
+      sudo apt-get -y install gcc
+      sudo apt-get -y install unzip
+      sudo apt-get install -y python-pip python
+      sudo apt-get install -y psmisc net-tools
+      sudo apt-get install -y git
+      sudo apt-get install -y gfortran
+      sudo apt-get install -y libjpeg-dev
+      echo 'export DISPLAY=127.0.0.1:0.0' >> .profile
+
+#. Install SCT
+
+   Download SCT:
+
+   .. code-block:: sh
+
+      git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+      cd sct
+
+   To select a `specific release <https://github.com/neuropoly/spinalcordtoolbox/releases>`_, replace X.Y.Z below with the proper release number. If you prefer to use the development version, you can skip this step.
+
+   .. code-block:: sh
+
+      git checkout X.Y.Z
+
+   Install SCT:
+
+   .. code:: sh
+
+      yes | ./install_sct
+
+#. Install FSLeyes
+
+   FSLeyes is the default viewer for NIfTI image in SCT.
+   
+   Install the C/C++ compilers required to use wxPython:
+   
+   .. code-block:: sh
+
+           sudo apt-get install build-essential
+           sudo apt-get install libgtk2.0-dev libgtk-3-dev libwebkitgtk-dev libwebkitgtk-3.0-dev
+           sudo apt-get install libjpeg-turbo8-dev libtiff5-dev libsdl1.2-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libnotify-dev freeglut3-dev
+           
+   Install wxPython using conda:
+   
+   .. code-block:: sh
+
+           yes | conda install -c anaconda wxpython
```

Comment:
is conda already installed by default in WSL? if not we should instead activate SCT's conda env, no?

Reply:
No, conda is not installed by default in WSL. My understanding is that conda is installed and the venv_sct activated in the previous steps in `install_sct ` [here](https://github.com/neuropoly/spinalcordtoolbox/blob/d553dac89c3e058727ff9121bd98269d172db788/install_sct#L510).

Reply:
indeed, but conda is then deactivated:

https://github.com/neuropoly/spinalcordtoolbox/blob/d553dac89c3e058727ff9121bd98269d172db788/install_sct#L602

so we would need to reactivate it

Reply:
@lrouhier since I don't have a WSL environment we really need your help here.

Reply:
I have now explicitly added info about activating SCT's conda environment. I would need someone to test it please.

Reply:
Conda is not installed by default. I can test the new instructions today.

Reply:
I tested the new instruction everything seems to work on my end.

Reply:
> After a check with another WSL solution: These instructions are made for WSL1 (maybe it needs to be added)
One of the differences between WSL1 and WSL2 is that WSL2 address for the display will not be 127.0.0.1:0.0.
@gaspardcereza, I think you have WSL2 do you confirm ?

Based on:
- some of the comments above regarding the complexity of WSL2
- the fact that SCT's core dev team is not familiar with WSL/WSL2
- our core dev team is small (need to prioritize)
- WSL1 is more distributed than WSL2
I would suggest to drop support for WSL2 for now.